addon/orm/associations/association.js

```diff
@@ -49,15 +49,7 @@ export default class Association {
     return this.opts.polymorphic;
   }
 
-  get isHasMany() {
-    return this.constructor.name === 'HasMany';
-  }
-
-  get isBelongsTo() {
-    return this.constructor.name === 'BelongsTo';
-  }
-
   get identifier() {
-    return this.isHasMany ? `${camelize(singularize(this.key))}Ids` : `${camelize(this.key)}Id`;
+    return `${camelize(this.key)}Id`;
```

Comment:
could we move this to `BelongsTo` and do something here like `throw 'Subclasses of Association must implement identifier'`?